### DISCUSSION TITLE
Title Header Hidden on Infinite Scroll

### DISCUSSION THREAD

**Discussion by hibikikuze4dan** [Difegue/LANraragi#574] [General]
I noticed that the Header that normally displays the comic title is hidden when in infinite scroll view.

https://i.imgur.com/hcCoNL8.png

But when you select either of the options to display or hide the header in the options, the header will reappear.

https://i.imgur.com/NFAb3Xc.png

**Comment by Difegue** [comment]
Hm this is super minor, but I'll try to think about fixing it. 😅

**Reply by hibikikuze4dan** [reply]
Lol, yeah, sorry to be a bother. Was just pointing it out for visibility's sake since I hadn't seen it brought up anywhere.

**Reply by Difegue** [reply]
Fixed 👍
